.github/workflows/reusable-test.yml

```diff
@@ -62,6 +62,10 @@ jobs:
 
   integration:
     if: ${{ inputs.environment-name=='Branch' }}
+    # limiting to one instance at a time as running these in parallel can cause
```

Comment:
why do we think running them in parallel can cause issues? we run every test against a different application?